pennylane_lightning/core/src/simulators/lightning_qubit/measurements/tests/Test_MeasurementsLQubit.cpp

```diff
@@ -521,6 +521,84 @@ TEMPLATE_PRODUCT_TEST_CASE("Probabilities", "[Measurements]",
                                        .margin(1e-7));
             }
         }
+        SECTION("5 qubits") {
+            constexpr std::size_t num_qubits = 5;
+            auto statevector_data =
+                std::vector<ComplexT>((1UL << num_qubits), {0.0, 0.0});
+            const std::vector<std::size_t> wires{0, 1, 2, 3, 4};
+            statevector_data[0] = {1.0, 0.0};
+            StateVectorT statevector(statevector_data.data(),
+                                     statevector_data.size());
+            Measurements<StateVectorT> Measurer(statevector);
+            SECTION("1 target") {
+                std::size_t target = GENERATE(0, 1, 2, 3, 4);
+                statevector.applyOperation("Hadamard", {target}, false);
+                auto probs = Measurer.probs({target}, wires);
+                CHECK_THAT(probs,
+                           Catch::Approx(std::vector<PrecisionT>(2, 1.0 / 2))
+                               .margin(1e-7));
+            }
+            SECTION("2 targets") {
+                std::size_t target0 = GENERATE(0, 1, 2, 3, 4);
+                std::size_t target1 = GENERATE(0, 1, 2, 3, 4);
```

Comment:
Validate all combinations of: Hadamard on either or both targets and probs on both targets.